packages/chats/src/BubblePlainText.tsx

```diff
@@ -153,24 +161,26 @@ function PlainTextContainer(
     const roundedCornerStyle = useBubbleRoundedCornerStyle(props, position);
     const actionString = getActionString(props);
 
+    const textLongPressHandler = useTextLongPressHandler()
+
+    const copyToClipboard = React.useCallback(()=>{
+        bubbleScaleAnimation(true);
+        props.text && Clipboard.setString(props.text);
+        textLongPressHandler && textLongPressHandler()
+        /**
+         * Maybe it's not the best place to run haptic
+         * but I don't want to put it in legacy package
+         * so left it here, until we make new share manager
+         */
+        runOnUI(hapticImpact)('medium');
+    },[props.text, textLongPressHandler, bubbleScaleAnimation])
+    
     return (
         <View style={containerStyle} onLayout={props.onLayout}>
             <TouchableWithoutFeedback
                 onPressOut={() => bubbleScaleAnimation()}
                 onPress={props.onTouchText}
-                onLongPress={() => {
-                    bubbleScaleAnimation(true);
-                    /**
-                     * Maybe it's not the best place to run haptic
-                     * but I don't want to put it in legacy package
-                     * so left it here, until we make new share manager
-                     */
-                    runOnUI(hapticImpact)('medium');
-                    UIShareManager.copyToClipboard(
-                        props.text,
-                        uiLocalized.MessageCopiedToClipboard,
-                    );
-                }}
+                onLongPress={copyToClipboard}
```

Comment:
Btw, It's not necessary that we'd like to copy the text to clipboard since it's up to parent how to behave IMHO, maybe it's better to rename the function to something like `handleLongPress` or alike